cpp/src/Ice/Instance.h

```diff
@@ -102,22 +102,22 @@ namespace IceInternal
         Ice::ObjectPtr findAdminFacet(std::string_view);
         Ice::FacetMap findAllAdminFacets();
```

Comment:
Not clear why is not added to this find functions.

Reply:
Same here: non-const.